### ISSUE TITLE
Is there a possibility to use this without a map? 

### ISSUE THREAD

**Issue by bowis** [soal/vue-mapbox#243]
MapBox has the example at: https://docs.mapbox.com/mapbox-gl-js/example/mapbox-gl-geocoder-no-map/ where the input is used without a map, is this in any way possible with this plugin?

**Comment by thopiddock** [comment]
Yes. It's **literally** the example. Forgive me, but I'm not sure what you're trying to do that is any different to what you see in that link.

You just import the script using the CDN link as in the example, then bind a new geocoder instance to a div of your choice and handle the event for the result in whatever way you want.

**Comment by bowis** [comment]
@thopiddock I meant with the use of the VueMapbox plugin, so that I can use the components from this plugins. I am able to read and understand hte given example, i merely was looking for another way to handle the event handlers in a more Vue manner.

**Comment by thopiddock** [comment]
Aha! That makes sense:


https://github.com/soal/vue-mapbox-geocoder

**Comment by hamzaafzal9009** [comment]
@Bowis are you able to use geocoder without the map?

**Comment by bowis** [comment]
> @Bowis are you able to use geocoder without the map?

yes, I was
